tests/api2/test_iscsi_auth_network.py

```diff
@@ -97,7 +97,7 @@ def configure_iscsi_service():
             call('service.stop', 'iscsitarget')
 
 
-@functools.cache
+@functools.lru_cache
```

Comment:
```suggestion
@functools.lru_cache(maxsize=None)
```

This does what `functools.cache` does in python 3.9

Reply:
aha... fixing